prow/plugins/trigger/ic.go

```diff
@@ -70,7 +70,7 @@ func handleIC(c client, ic github.IssueCommentEvent) error {
 
 	for _, job := range requestedJobs {
 		c.Logger.Info("Starting %s build.", job.Name)
-		if err := build(c, job, *pr); err != nil {
+		if err := lineStartJob(c.KubeClient, job.Name, *pr); err != nil {
```

Comment:
I think it's a reasonably common pattern in Go. Doing proper dependency injection is verbose here, and it's only "global" to the package. See, for example, https://golang.org/src/net/hook.go .